tests/unit/components/UI/Text/URL.spec.js

```diff
@@ -0,0 +1,81 @@
+import { mount } from "@vue/test-utils";
+import URL from "@/components/UI/Text/URL";
+import Tooltip from "primevue/tooltip";
+
+// mock document.execCommand
+document.execCommand = jest.fn();
+
+describe("URL.vue", () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(URL, {
+      global: {
+        directives: {
+          tooltip: Tooltip,
+        },
+      },
+    });
+  });
+
+  it("should render with default values", () => {
+    expect(wrapper).toBeTruthy();
+  });
+
+  it("link renders correctly", async () => {
+    const link = "testLink";
+    await wrapper.setProps({ link: link });
+
+    expect(wrapper.get('[data-test="link"]').text()).toBe(link);
+  });
+
+  it("link text class is set correctly", async () => {
+    const link = "testLink";
+    const linkClass = "w-12 h-12";
+    await wrapper.setProps({
+      link: link,
+      urlStyleClass: linkClass,
+    });
+
+    expect(wrapper.get('[data-test="link"]').classes()).toEqual(
+      expect.arrayContaining(["w-12", "h-12"])
+    );
+  });
+
+  it("link text underlining works correctly", async () => {
+    const link = "testLink";
+    await wrapper.setProps({
+      link: link,
+      isUnderlined: true,
+    });
+
+    expect(wrapper.get('[data-test="link"]').classes()).toEqual(
+      expect.arrayContaining(["border-b-2", "border-yellow-500"])
```

Comment:
This hardcoding might cause pain later on. Isn't it better to actually retrieve the underline class from the component, save it in a variable and then assert the tested . classes () with that variable? This will help in long term because we keep changing the CSS classes often. Everytime we'll need to change the test cases. We can access the computed properties from a wrapper and hence we should be able to access the class which is the underline class here